### PR TITLE
added hh network config RSK testnet and mainnet

### PR DESCRIPTION
**Pull request by tjcloa**

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -51,6 +51,16 @@ module.exports = {
 	},
 	networks: {
 		hardhat: {},
+		rskTestnet: {
+			url: "https://public-node.testnet.rsk.co/",
```
[the remark this answers is not in the document]
Comment (tjcloa) [reply hardhat.config.js:55]: added Sovryn nodes

File: hardhat.config.js [hunk 2]
```diff
@@ -51,6 +51,16 @@ module.exports = {
...
+			accounts: { mnemonic: "brownie", count: 10 },
+			network_id: 31,
+			confirmations: 4,
+		},
+		rskMainnet: {
+			url: "https://public-node.rsk.co/",
```
[the remark this answers is not in the document]
Comment (tjcloa) [reply hardhat.config.js:61]: does it look good now?

**Review by powerhousefrank — APPROVED** [review]
LGTM

Just a suggestion, you could add the `gasMultiplier` to the testnets, and set it to 1.2 or higher.

Also, add a `timeout` into all the networks, but you can keep it commented, just so that if someone needs to have an increased timeout, he can quickly make the change, without going through the hh config details.

**Comment by powerhousefrank** [comment]
Yes, it can be merged now. Good work!

**Pull request merged**